dev_docs/key_concepts/encrypted_saved_objects.mdx

```diff
@@ -0,0 +1,175 @@
+| Add a new attribute                          | Yes        | N/A     | Implement a Model Version and wrap it in a call to `createModelVersion`. Implement Model Version changes as needed. Implement a `forwardCompatibility` schema and ignore unknowns if the previous version of Kibana will not be able to tolerate additions to the attribute.                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Add an existing attribute to AAD             | No         | No->Yes | This is not currently supported. Existing attributes that are in use and populated with data cannot be added to AAD. The previous version of Kibana will never be able to successfully perform decryption in this case.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| Remove an existing attribute                 | No         | No      | Implement the Model Version as needed. There is no need to wrap the Model Version with `createModelVersion` because AAD is unaffected. If the previous version of will not be able to tolerate the missing attribute, this will require 2 Serverless release stages. `Release 1`: update all business logic to handle this type without the attribute that will be removed. `Release 2`: implement the Model Version as described.                                                                                                                                                                                                                                                                         |
```

Comment:
__nit:__ I see in other cases we use `a Model Version`, and here `the Model Version` - is it intentional? 🙂 

> previous version of will 

Previous version of ... __Kibana__? (here and in a few places below)

Reply:
Some of them were intentional, but not all. I cleaned it up! And yes, I was missing "Kibana" in a few places due to a copy/paste error.